contracts/interfaces/IRFQ.sol

```diff
@@ -4,11 +4,16 @@ pragma abicoder v2;
 
 import "./IStrategyBase.sol";
 import "../utils/RFQLibEIP712.sol";
+import "../utils/SpenderLibEIP712.sol";
 
 interface IRFQ is IStrategyBase {
     function fill(
         RFQLibEIP712.Order memory _order,
+        SpenderLibEIP712.SpendWithPermit memory _makerAssetPermit,
+        SpenderLibEIP712.SpendWithPermit memory _takerAssetPermit,
```

Comment:
We don't need to include all params from taker/maker in the call data since most of them were already in `_order`.

Reply:
And also we can reuse `salt` and `expiry` in `_order`.

Reply:
```solidity
function fill(
    RFQLibEIP712.Order memory _order,
    bytes memory _mmSignature,
    bytes memory _userSignature,
    bytes memory _makerAssetPermitSig,
    bytes memory _takerAssetPermitSig
)
```

Reply:
OK! Take the RFQ contract as an example,
the Parameters and the Signature required by the user to complete the spendWithPermit type.